spec/v2/index.bs

```diff
@@ -621,21 +621,23 @@ Advisement:
       <tr>
         <td><dfn>primaryDataShare</dfn> : [=Percent=]
         <td>Number
-        <td>O*
+        <td>O
         <td>
-          The share of primary data in percent. See the [=Pathfinder Framework=] Section 4.2.2.
+          The share of primary data in percent. See the [=Pathfinder Framework=] Sections 4.2.1 and 4.2.2, Appendix B.
 
-          For reporting periods ending before 2025 and if this property is present, the property <{CarbonFootprint/dqi}> MUST NOT be defined.
+          For reporting periods ending before the beginning of year 2025, at least property <{CarbonFootprint/primaryDataShare}> or propery <{CarbonFootprint/dqi}> MUST be defined.
 
-          For reporting periods including the beginning of year 2025 or after, this property MUST be defined.					
+          For reporting periods including the beginning of year 2025 or after, this property MUST be defined.	
       <tr>
         <td><dfn>dqi</dfn> : <{DataQualityIndicators}>
         <td>Object
-        <td>O*
+        <td>O
```

Comment:
Same comment as above - why should this be changed from 0* to 0?

Reply:
Likewise addressed via a464d9503a2e5cb83e9661cdb14940ad94568900